### PR TITLE
Update server

### PR DESCRIPTION
**Pull request by predragnikolic**

### CODE REVIEW

File: LSP-tailwindcss.sublime-settings [hunk 1]
```diff
@@ -19,6 +24,7 @@
 		"tailwindCSS.lint.invalidVariant": "error",
 		"tailwindCSS.lint.recommendedVariantOrder": "warning",
 		"tailwindCSS.experimental.classRegex": [],
+		"tailwindCSS.inspectPort": null
```
Comment (rchl) [inline LSP-tailwindcss.sublime-settings:27]: This one is handled by vscode extension so won't do anything for us.

File: language-server/package.json [hunk 2]
```diff
@@ -119,6 +119,18 @@
           "default": {},
           "markdownDescription": "Enable features in languages that are not supported by default. Add a mapping here between the new language and an already supported language.\n E.g.: `{\"plaintext\": \"html\"}`"
         },
+        "tailwindCSS.files.exclude": {
```
Comment (rchl) [inline language-server/package.json:122]: This one also seems like handled in the extension.

Comment (rchl) [reply language-server/package.json:122]: Actually maybe not: https://github.com/tailwindlabs/tailwindcss-intellisense/blob/9bc82821c7dfef20db694f5fd255869b766c01f1/packages/tailwindcss-language-server/src/util/isExcluded.ts#L13

Though I see it also used in the extension. I wonder how much we are missing without the extension glue code.

Comment (predragnikolic) [reply language-server/package.json:122]: Yes, it seems like this setting is used both on the language server and in the VS code extension,
For now I would leave this setting.

**Review by rchl — APPROVED** [review]

**Pull request merged**